firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -71,15 +64,13 @@ public List<Component<?>> getComponents() {
         Component.builder(FirebasePerfEarly.class)
             .name(EARLY_LIBRARY_NAME)
             .add(Dependency.required(FirebaseApp.class))
-            .add(Dependency.required(FirebaseSessions.class))
             .add(Dependency.optionalProvider(StartupTime.class))
             .add(Dependency.required(uiExecutor))
             .eagerInDefaultApp()
             .factory(
                 container ->
                     new FirebasePerfEarly(
                         container.get(FirebaseApp.class),
-                        container.get(FirebaseSessions.class),
```

Comment:
I'm not sure I understand why perf is losing its sessions dep here. Is it just no longer needed early, but is used later?

Reply:
This was intentional. We removed perf integration for now to cut scope.